evaluate/task/symflower-fix.go

```diff
@@ -0,0 +1,50 @@
+package task
+
+import (
+	"context"
+	"time"
+
+	pkgerrors "github.com/pkg/errors"
+	"github.com/symflower/eval-dev-quality/evaluate/metrics"
+	"github.com/symflower/eval-dev-quality/language"
+	"github.com/symflower/eval-dev-quality/log"
+	"github.com/symflower/eval-dev-quality/tools"
+	"github.com/symflower/eval-dev-quality/util"
+)
+
+// symflowerFix runs the "symflower fix" command and returns the corresponding assessments.
+func symflowerFix(logger *log.Logger, modelAssessment metrics.Assessments, repositoryPath string, language language.Language) (assessments metrics.Assessments, problems []error, err error) {
```

Comment:
Please split this function to have the symflower fix command call separated form the assessment combining logic. And consider moving the assessment combination logic into the assessment package where it can easily be unit tested (i.e. if the model had this assessment and the fixed result had that assessment then result should be...).